lib/minfraud/components/credit_card.rb

```diff
@@ -77,7 +86,8 @@ class CreditCard < Base
       def initialize(params = {})
         @bank_phone_country_code  = params[:bank_phone_country_code]
         @issuer_id_number         = params[:issuer_id_number]
-        @last_4_digits            = params[:last_4_digits]
+        @last_digits              = params[:last_digits] || params[:last_4_digits]
+        @last_4_digits            = @last_digits
```

Comment:
Do we need to set this?

Reply:
I believe that would break backwards compatibility. A test enforcing this was added here: https://github.com/maxmind/minfraud-api-ruby/pull/60/files#diff-bbca7dbf04fa02ed9c465f45c9a6d0ba19abe6301af1cd86dd8e72f6926ca938R80

Reply:
Hmm yeah, good point! I think this will cause us to send both `last_digits` and `last_4_digits` to the API though as that's based on the instance variables. Will that be handled ok on the API side and not generate warnings? Ideally we wouldn't send both I think.

Reply:
The API would warn, yes. I suppose I can instead turn `last_4_digits` into a method though, which I assume would maintain backwards compatibility while only sending `last_digits` to the API

Reply:
I see. Yes, that may be best then. I think we'd need a setter method as well as a reader!